integration-tests/integration/coordinator/add-involved-staff.spec.js

```diff
@@ -82,10 +84,9 @@ context('A use of force coordinator can add involved staff', () => {
     viewStatementsPage = ViewStatementsPage.verifyOnPage()
     viewStatementsPage.return().click()
 
-    const allIncidentsPage = AllIncidentsPage.verifyOnPage()
-    allIncidentsPage.getTodoRows().should('have.length', 1)
-    allIncidentsPage.getNoCompleteRows().should('exist')
```

Comment:
no longer testing that the previous report is complete... this was the idea around adding a get report count DB method.

Reply:
testing the report is complete by navigating to the completed incidents page and checking it is empty

Reply:
now you have added the db method you can use it here?

Reply:
DB method implemented to get a count of the reports that have a status of COMPLETE and asserted that to be zero as the test is to move the report from complete to incomplete